src/FormBuilder/resources/js/form-builder/src/components/settings/ControlForPopover/index.tsx

```diff
@@ -47,3 +46,22 @@ export default function ControlForPopover({
         </BaseControl>
     );
 }
+
+export function EditIcon() {
```

Comment:
Can you please move this into its own file within the same folder and import into here

Reply:
Yes. Resolved: 3b03779129e7f007873fea44fceb87a58f81f540